sqls/test3.sql

```diff
@@ -0,0 +1 @@
+  SELECT a  +  b FROM tbl;
```

Comment:
🚫 **[sqlfluff]** <[LT02](https://docs.sqlfluff.com/en/stable/rules.html#rule-LT02)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[layout.indent] - First line should not be indented.
```suggestion
SELECT a  +  b FROM tbl;
```